### PR TITLE
ox status: dense knowledge-bubble listing + bump to 0.9.1

### PR DESCRIPTION
**Pull request by rsnodgrass**
## What this PR ships

Two related pieces of work on the `ox status` surface, plus a version bump.

- **Version → 0.9.1** across the canonical source and all mirrors (`internal/version/version.go`, plugin manifests, `docs/package.json`, `CHANGELOG`). `make verify-version` passes.
- **`ox status` knowledge-bubble redesign** — replaces the count-only line + duplicated "Other Team Contexts" section with one owner-grouped tree.
- **CLI reference regeneration** — picks up accumulated drift (missing `scout`, `plan lint`, `kb config` pages; sidebar ordering) plus the new `status --verbose` flag.

## The problem (status before)

The knowledge-bubble section showed a bare count, then listed every team **again** under a separate header with a full filesystem path on each row:

```
Knowledge bubbles   9 (8 team, 1 repo)

Other Team Contexts
───────────────────
Team                SageOx Internal
  Visibility        private (✓ member)
  Path              /Users/ryan/.local/share/sageox/sageox.ai/teams/team_xlcr6yzpec
  Status            ⚠ 6 uncommitted
... (4 lines per team, full path every time, teams duplicated with the count)
```

- No real per-bubble status in the "Knowledge bubbles" line — just a tally.
- Same teams duplicated between the count and the list.
- The long data-dir prefix repeated on every row. Does not scale to dozens of bubbles.

## What this PR ships (status after)

An owner-grouped tree. Each owner is a parent node `@slug  (Display Name)`; its knowledge bubbles nest beneath under solid tree glyphs, each with a compact, color-coded status:

```
Knowledge bubbles  13 total · 7 owners
─────────────────
  on disk  ~/.local/share/sageox/sageox.ai/teams/

@sageox-internal  (SageOx Internal)
└─ (team)                ⚠ 6 uncommitted
@dry-run-ice-cream-2026-05-01  (Dry Run - Ice Cream 2026-05-01)
└─ (team)                ✓ 3d
@open-docs  (Open Docs)
└─ (team)                ✓ 5m   PUBLIC
…
```

- **Owner parent node** — `@slug  (Display Name)`. The owner's own context nests as `(team)`; a future multi-bubble owner reads `├─ #sageox-mono (repo)`, `└─ #marketing (custom)` with **no layout change**.
- **Slugs never truncated** — the full identifier always shows.
- **Status with the bubble** — `✓ <age>` when clean (crisp `now`/`5m`/`2h`/`3d`/`2w`), `⚠ N uncommitted` when dirty, red `⚠ rebase wedged` when stuck. Needs-attention owners sort to the top.
- **PUBLIC-only visibility** — nearly every bubble is private, so private is the silent default (no marker); only `PUBLIC` is flagged, in bold + the public accent token, so it pops.
- **Shared on-disk prefix printed once**, never per row. `--verbose`/`-v` reveals opaque IDs and full paths.
- Fits 80 columns, semantic theme styles only (no raw colors), `NO_COLOR`-safe.

## Data flow

```mermaid
flowchart LR
  CLI["ox status"] --> CTC["cloud team contexts (name, slug, visibility)"]
  CLI --> MRG["kb merger (headline total)"]
  CTC --> ROWS["build otherTeamRow + git status per checkout"]
  ROWS --> SORT["sort: needs-attention first, then name"]
  SORT --> REND["render owner tree, prefix once, PUBLIC-only"]
  MRG --> REND
```

## Design notes / deliberate scope

- The listing is driven by existing **cloud team-context** data (already carries `slug`/visibility/status via `RepoInfo.Slug`); the kb merger supplies only the headline total. No new data plumbing.
- The **multi-bubble-per-owner** nesting is forward-looking — no source exposes more than one bubble per owner today, so each owner renders one `(team)` child. The tree layout absorbs additional bubbles unchanged when the API grows them.
- The ledger + primary-team blocks in **Project Status** are left untouched — they carry many fragile edge-case branches (not-cloned, bootstrapping, wedged, viewer) and are not the density problem.

## Test Plan

- `make verify-version` to "All versions match: 0.9.1"
- `go build ./... && go vet ./cmd/ox/ ./internal/status/` clean
- `make lint` to 0 issues
- `go test ./cmd/ox/ ./internal/status/` green, incl. new tests:
  - `bubblesCountSummary`, `sortOtherBubbleRows` (attention-first), `renderOtherBubbleRows` (owner node + nested `(team)`, full slug / no ellipsis, prefix-once, PUBLIC flagged + private silent, `--verbose` reveals paths), `renderBubbleStatus` variants, `renderSlugRef`, `CompactAge`
- Manual: `ox status`, `ox status --verbose`, `COLUMNS=80 NO_COLOR=1 ox status` (max line 63 cols)

Co-Authored-By: SageOx <ox@sageox.ai>

## Summary by CodeRabbit

* **New Features**
  * Denser "Knowledge bubbles" in ox status with owner-grouped rows, attention-first sorting, compact color-coded freshness/status, and a -v/--verbose mode to reveal opaque IDs and full on-disk paths.
  * Added ox plan lint, ox scout, and ox kb config commands; ox query gains a --local offline search option.
* **Documentation**
  * Many CLI reference pages added/updated and sidebar ordering refreshed.
* **Tests**
  * New tests covering bubbles rendering and compact age formatting.
* **Chores**
  * Version bumped to 0.9.1.

### CODE REVIEW

File: cmd/ox/status.go [hunk 1]
```diff
@@ -890,29 +892,82 @@ func renderGitReposSection(localCfg *config.LocalConfig, projectRoot string, dae
...
-		for _, entry := range otherCloudTCs {
-			hasAnyTeams = true
-			renderCloudTC(entry.info)
-		}
-		for _, entry := range otherDetailTCs {
-			hasAnyTeams = true
-			renderDetailTC(entry.info)
-		}
+		b.WriteString(renderOtherBubbleRows(otherRows, daemonStatus.IsBootstrapping(), verbose))
 	}
```
[the remark this answers is not in the document]
Comment (rsnodgrass) [reply cmd/ox/status.go:980]: Fixed in b48c68c7 — when bubbles exist but no team contexts are cloned locally, the section now shows a muted hint (`run 'ox kb list' to see all bubbles`) instead of an empty header + separator block.

File: cmd/ox/status_bubbles.go [hunk 2]
```diff
@@ -256,3 +260,144 @@ func newDefaultStatusBubblesMerger(projectRoot string) statusBubblesMerger {
...
+func renderOtherBubbleRows(rows []otherTeamRow, bootstrapping, verbose bool) string {
+	if len(rows) == 0 {
+		return ""
+	}
+	var b strings.Builder
+
+	// shared on-disk prefix, printed once instead of per row
+	base := filepath.Dir(rows[0].path)
+	b.WriteString(statusLabelStyle.Render("  on disk"))
+	b.WriteString(statusMutedStyle.Render(shortenHome(base) + string(os.PathSeparator)))
```
[the remark this answers is not in the document]
Comment (rsnodgrass) [reply cmd/ox/status_bubbles.go]: Fixed in b48c68c7 — `base` is now `commonBubbleBase(rows)`, the deepest directory shared by ALL rows (not the attention-sorted `rows[0]`). Covered by TestCommonBubbleBase, including a mixed teams/ + kb/ case.

File: cmd/ox/status_bubbles_test.go [hunk 3]
```diff
@@ -397,3 +398,118 @@ func stripANSIBubbles(s string) string {
...
+
+	dirty := stripANSIBubbles(renderBubbleStatus(gitRepoStatus{Exists: true, UncommittedCount: 18}, true, false))
+	assert.Equal(t, "⚠ 18 uncommitted", dirty)
+
+	wedged := stripANSIBubbles(renderBubbleStatus(gitRepoStatus{Exists: true, RebaseInProgress: true}, true, false))
+	assert.Equal(t, "⚠ rebase wedged", wedged)
+
+	notCloned := stripANSIBubbles(renderBubbleStatus(gitRepoStatus{}, false, false))
+	assert.Equal(t, "⚠ not cloned", notCloned)
+}
```
[the remark this answers is not in the document]
Comment (rsnodgrass) [reply cmd/ox/status_bubbles_test.go:525]: Fixed in b48c68c7 — added a diverged case to TestRenderBubbleStatus: `gitRepoStatus{Exists:true, AheadCount:2, BehindCount:1}` → `⚠ diverged`, completing IsWedged() coverage.

File: cmd/ox/release_notes.md [hunk 4]
```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+
+- **`ox status` knowledge bubbles, denser and more useful** — the bubble section used to print a bare count (`9 (8 team, 1 repo)`) and then repeat every team again under "Other Team Contexts" with a full filesystem path on each row. It's now a single, scannable listing: each owner shown as `@slug` with a compact, color-coded freshness status (`✓ 2h`, `⚠ 6 uncommitted`), the shared on-disk prefix printed once instead of per row, and teams that need attention (uncommitted or wedged) sorted to the top. Add `--verbose`/`-v` to reveal the opaque IDs and full paths when you need them.
+
```
[the remark this answers is not in the document]
Comment (rsnodgrass) [reply cmd/ox/release_notes.md]: Fixed in b48c68c7 — moved the `### Changed` ox status entry under `## [0.9.1] - 2026-06-08`, so the redesign is attributed to the 0.9.1 release.
